docs/web.md

```diff
@@ -0,0 +1,170 @@
+# Web Frameworks
+
+Since HTTP logging is a primary use case, Pino has first class support for the Node.js 
+web framework ecosystem.
+
++ [Pino with Fastify](#fastify)
++ [Pino with Express](#express)
++ [Pino with Hapi](#hapi)
++ [Pino with Restify](#restify)
++ [Pino with Koa](#koa)
++ [Pino with Node core `http`](#http)
+
+<a id="fastify"></a>
+## Pino with Fastify
+
+The Fastify web framework comes bundled with Pino by default, simply set Fastify's
+`logger` option to `true` and use `reply.log` for log messages that correspond 
+to each individual request:
+
+```js
+const fastify = require('fastify')({
+  logger: true
+})
+fastify.get('/', async (request, reply) => {
+  reply.type('application/json').code(200)
+  reply.log.info('something')
+  return { hello: 'world' }
+})
+```
+
+See the [fastify documentation](https://www.npmjs.com/package/pino-http#pinohttpopts-stream) for more information.
+
+<a id="express"></a>
+## Pino with Express
+
+```sh
+npm install --save express-pino-logger
+```
+
+```js
+const app = require('express')()
+const pino = require('express-pino-logger')()
+
+app.use(pino)
+
+app.get('/', function (req, res) {
+  req.log.info('something')
+  res.send('hello world')
+})
+
+app.listen(3000)
+```
+
+See the [express-pino-logger readme](http://npm.im/express-pino-logger) for more info.
+
+<a id="hapi"></a>
+## Pino with Hapi
+
+```sh
+npm install --save hapi-pino
+```
+
+```js
+'use strict'
+
+const Hapi = require('hapi')
+
+const server = new Hapi.Server()
+server.connection({ port: 3000 })
+
+server.route({
+  method: 'GET',
+  path: '/',
+  handler: function (request, reply) {
+    request.logger.info('In handler %s', request.path)
+    return reply('hello world')
+  }
+})
+
+server.register(require('hapi-pino'), (err) => {
+  if (err) {
+    console.error(err)
+    process.exit(1)
+  }
+
+  server.logger().info('another way for accessing it')
+
+  // Start the server
+  server.start((err) => {
+    if (err) {
+      console.error(err)
+      process.exit(1)
+    }
+  })
+})
+```
+
+See the [hapi-pino readme](http://npm.im/hapi-pino) for more info.
+
+<a id="restify"></a>
+## Pino with Restify
+
+```sh
+npm install --save restify-pino-logger
+```
+
+```js
+const server = require('restify').createServer({name: 'server'})
+const pino = require('restify-pino-logger')()
+
+server.use(pino)
+
+server.get('/', function (req, res) {
+  req.log.info('something')
+  res.send('hello world')
+})
+
+server.listen(3000)
+```
+
+See the [restify-pino-logger readme](http://npm.im/restify-pino-logger) for more info.
```

Comment:
Does this work with latest restify?

Reply:
I don't know, need to check. If not we'll need to update restify-pino-logger. 

In any case, this won't affect the docs

Reply:
of course it would affect the docs what am I talking about. 

anyway, it does, the syntax since restify v4 is largely unchanged for initialization

updated dev dep in restify-pino-logger from v4 to v7, tests pass and example works

https://github.com/pinojs/restify-pino-logger/commit/343930cc9696fcf0d1c7b2600757f718ce11c3e0